docs/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "twill-3x-docs",
+  "version": "1.0.0",
+  "description": "## Install",
+  "main": "tailwind.config.js",
+  "author": "Area 17",
+  "license": "ISC",
+  "dependencies": {
+    "@area17/a17-tailwind-plugins": "^3.7.0",
```

Comment:
Can we maybe keep these in the dev-dependencies in the main package.json to avoid needing to install npm in 2 locations? Or did that not work?

Reply:
hey @kylegoines  I will take care of this and merge.